spring-batch-docs/asciidoc/domain.adoc

```diff
@@ -70,6 +115,44 @@ functionality on top of `Job`, however the batch
 namespace abstracts away the need to instantiate it directly. Instead, the
 `<job>` tag can be used:
```

Comment:
The text needs to be updated as well since it talks about the `<job` tag and yet we show java config.